data-loader/cli/src/test/java/com/scalar/db/dataloader/cli/util/DirectoryUtilsTest.java

```diff
@@ -44,13 +48,25 @@ void validateOrCreateTargetDirectory_DirectoryDoesNotExist_CreatesDirectory()
   @Test
   void validateOrCreateTargetDirectory_DirectoryNotWritable_ThrowsException() throws IOException {
     Path readOnlyDirectory = Files.createDirectory(Paths.get(tempDir.toString(), "readOnlyDir"));
-    readOnlyDirectory.toFile().setWritable(false);
 
+    // Try to make it read-only using POSIX if supported
```

Comment:
Seems like the cause of the test failure is still unclear, so I think we shouldn't merge this kind of ad-hoc workaround.

I ran `./gradlew :data-loader:cli:build --no-daemon --stacktrace` without this workaround with the latest `data-loader-ci.yaml`, but it didn't fail
https://github.com/scalar-labs/scalardb/actions/runs/15702570827/job/44240661813. I guess the updated workflow file may have resolved the issue. Can you try without this workaround again just in case?

If it still fails, I think the unit tests are already executed in `ci.yaml`'s `check` job, so we can skip the unit tests in `data-loader-ci.yaml` by using `./gradlew -x test` like `./gradlew -x test :data-loader:cli:build`.

Reply:
@komamitsu san (cc @ypeckstadt ),
Sorry I missed this. You are right about the tests already executing in check job. And as I mentioned, since the test failure only occurs on my local environment with act, it might be some issue with my environment and we can ignore that.
Thank you for pointing this out.

Reply:
The change above was reverted by @ypeckstadt san in this commit [6def419](https://github.com/scalar-labs/scalardb/pull/2761/commits/6def4199fb1a36560a0d45aec3f8b277d9c147eb).